docs/library/cn0363/index.rst

```diff
@@ -0,0 +1,21 @@
+.. _cn0363_lib:
+
+CN0363
```

Comment:
Normally, we don't do it like this, to add "IP" to the title, but maybe it can be confusing when there is a project called CN0363 and also an IP.

```suggestion
CN0363 IP
```